toolbox/CMakeLists.txt

```diff
@@ -244,6 +253,10 @@ else()
     target_link_libraries(WBToolbox ${LINKED_LIBRARIES} ${Matlab_LIBRARIES})
 endif()
 
+# Remote Inverse Kinematics requires C++11
+set(CXX_STANDARD_REQUIRED ON)
+set_property(TARGET WBToolbox PROPERTY CXX_STANDARD 11)
```

Comment:
Eventually (when all the compiler that we support default to use C++11) we will want to drop this to avoid forcing the compiler to use an older language version (for example GCC 6.0 will default to C++14, and I don't know what will happen by forcing it to 11).